pomodoro-cli.sh

```diff
@@ -84,40 +105,43 @@ function display_help() {
   echo "    break              Minutes of break until focus      | Default = 5"
   echo "    long_break         Minutes of long break until focus | Default = 15"
   echo "    breaks_until_long  Number of breaks until long break | Default = 4"
+  echo "    apps_to_kill       ',' separated list of apps to kill| Default = none"
 }
 
 ### Controls the application flow, parse arguments, show the countdown and notifications
 ## Params: focus_minutes {Number} - Ammount of minutes to last a focus period
 ## Params: break_minutes {Number} - Ammount of minutes to last a break period
 ## Params: long_break_minutes {Number} - Ammount of minutes to last a long break period
 ## Params: breaks_until_long {Number} - Ammount of breaks until a long break period starts
+## Params: apps_to_kill {String} - Comma separated list of apps to kill
 ## Returns: {Void}
 function main() {
     focus_minutes=$1
     break_minutes=$2
     long_break_minutes=$3
     breaks_until_long=$4
+    apps_to_kill=$5
 
     focus_seconds=$(minutes_to_seconds "$focus_minutes")
     break_seconds=$(minutes_to_seconds "$break_minutes")
     long_break_seconds=$(minutes_to_seconds "$long_break_minutes")
 
 
-    display_summary "$focus_minutes" "$break_minutes" "$long_break_minutes" "$breaks_until_long"
+    display_summary "$focus_minutes" "$break_minutes" "$long_break_minutes" "$breaks_until_long" "$apps_to_kill"
 
     while true; do
       for (( i=1; i<=breaks_until_long; i++ )); do
-	countdown "$focus_seconds" "FOCUS TIME" 
+	countdown "$focus_seconds" "FOCUS" "$apps_to_kill" "FOCUS TIME"
```

Comment:
```suggestion
	countdown "$focus_seconds" "FOCUS TIME"
```
No need to mention that these apps are killed; Like the user gave details - he knows - plus that will remind him again that minecraft is closed and it might trigger him/her to play.

Reply:
I require to take the `$apps_to_kill` as a parameter as I am checking if any of them are opened inside the countdown function.

Reply:
making this change